nextjs-metrics-viewer/app/register/page.tsx

```
@@ -0,0 +1,225 @@
+"use client";
+
+import React, { useState } from "react";
+import { useRouter } from "next/navigation";
+import { Icons } from "@/components/icons";
+import type { SVGProps } from "react";
+import type { OverridableComponent } from "@mui/material/OverridableComponent";
+import type { SvgIconTypeMap } from "@mui/material/SvgIcon";
+
+function createIconWrapper(
+    MUIIcon: OverridableComponent<SvgIconTypeMap<{}, "svg">>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)